taxi/forms.py

```diff
@@ -0,0 +1,29 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+
+class DriverCreateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number", "password",)
+
+
+class CarForm(forms.ModelForm):
+    class Meta:
+        model = Car
+        fields = ("model", "manufacturer", "drivers",)
+
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
```

Comment:
The `queryset` for the `drivers` field in `CarForm` is currently using `get_user_model().objects.all()`, which should be replaced with `Driver.objects.all()`. This change is crucial to ensure that the queryset correctly fetches `Driver` instances, allowing for proper driver assignment to cars.